lbz/resource.py

```diff
@@ -123,28 +125,23 @@ def __call__(self):
             resp.headers.update(self.resp_headers())
         return resp
 
-    def _check_for_wildcard(self):
-        org = self.request.headers.get("Origin")
-        for aorg in self._allowed_origins:
-            if org == aorg:
-                self._allowed_origins = [org]
-                break
-            if "*" in aorg:
-                serv, domain = aorg.split("*")
-                if org.startswith(serv) and org.endswith(domain):
-                    self._allowed_origins = [org]
-                    break
+    def _get_allowed_origins(self, origins):
+        if orig := self.request.headers.get("Origin"):
+            for aorg in origins:
+                if orig == aorg:
+                    return [orig]
+                if "*" in aorg:
+                    serv, domain = aorg.split("*")
```

Comment:
I understand `aorg`, but `serv` instead of `server` does not look good

Reply:
I don't think we should abbreviate variable names unless we have a good reason to do so - having the full names even for `aorg` and `orig` shouldn't be problematic (they are not too long too) 😉

Reply:
orig is only becouse You have origins next to it and having origins and origin in same line is missleading.